RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -24,28 +24,35 @@ public static class WebViewPageExtensions
 
         private static readonly Dictionary<string, ViewPlaceholder> _views = new Dictionary<string, ViewPlaceholder>();
 
-        public static IViewEngine ViewEngine
+        private static StringWriter _writer;
+
+        public static DummyViewEngine ViewEngine
         {
             get { return _viewEngine; }
         }
 
+        public static StringWriter Writer
```

Comment:
This is the only usage: 

    Mock.Get(httpContextBase.Response).Setup(c => c.Output).Returns(() => WebViewPageExtensions.Writer)

So main view gets the writer as Response.Output, and it's propagated to child views. Maybe it would be better to pass _in_ the writer instead of pulling it out? Is that what you mean by flowing it in? :)